modules/staking/source/source.go

```diff
@@ -12,4 +14,5 @@ type Source interface {
 	GetValidatorDelegations(height int64, validator string) ([]stakingtypes.DelegationResponse, error)
 	GetPool(height int64) (stakingtypes.Pool, error)
 	GetParams(height int64) (stakingtypes.Params, error)
+	GetValidatorSigningInfo(height int64, consAddr sdk.ConsAddress) (slashingtypes.ValidatorSigningInfo, error)
```

Comment:
I believe this should be placed inside the `SlashingSource` interface instead, since it relates to the `slashing` module